src/sock.c

```diff
@@ -358,3 +378,4 @@ int getpeer_information (int fd, char *ipaddr, char *string_addr)
         return getnameinfo ((struct sockaddr *) &sa, salen,
                             string_addr, HOSTNAME_LENGTH, NULL, 0, 0);
 }
+
```

Comment:
looks good! apart from this gratuitous whitespace change here. wanna try the squashing yourself? if not i can do it too.